templates/Template.NEP17.CSharp/NEP17.cs

```diff
@@ -23,15 +24,16 @@ public partial class NEP5 : SmartContract
         #region Notifications
         [DisplayName("Transfer")]
         public static event Action<UInt160, UInt160, BigInteger> OnTransfer;
+
+        [DisplayName("Payment")]
+        public static event Action<UInt160, BigInteger, object> Payment;
```

Comment:
```suggestion
        public static event Action<UInt160, BigInteger> Payment;
```
https://github.com/neo-project/proposals/pull/126#discussion_r530927491